libs/causality/src/lib/CausalAnalysisDashboard/CausalInsights.styles.ts

```diff
@@ -17,7 +17,8 @@ export const causalInsightsStyles: () => IProcessedStyleSet<ICausalInsightsStyle
     const theme = getTheme();
     return mergeStyleSets<ICausalInsightsStyles>({
       container: {
-        color: theme.semanticColors.bodyText
+        color: theme.semanticColors.bodyText,
```

Comment:
can we remove the custom color? This should be default, shouldn't it?

Reply:
This is for theming support